src/validators/tasks.py

```diff
@@ -127,6 +127,15 @@ async def register_validators(
         # not enough balance to register validators
         return None
 
+    # Check if there is enough ETH to register the specified minimum number of validators
+    if validators_count < settings.min_validators_required:
+        logger.info(
```

Comment:
Will be spamming on each block. Do we need it? I would like to decrease logging amount somehow.

Reply:
maybe log only in verbose mode